components/GothamistArticle.vue

```diff
@@ -442,6 +442,9 @@ export default {
           this.trackScrollDepth(threshold)
           this.currentlyWatching.splice(index, 1)
         }
+        if (this.scrollPercent >= 75) {
+          this.bannerOnscreen = true
+        }
```

Comment:
My only concern here is that it looks like this will just repeatedly fire and set `bannerOnScreen` to `true` over and over again as the user scrolls between 75%-100%, since it fires anytime the scroll percentage changes. It probably wouldn't have any effect on performance, but who knows.

If I'm understanding Matt's code correctly, the `threshold` gets hit on intervals of [0, 25, 50, 75, 100] and, once hit, it's removed from the array, so I think if you did something like `if (this.scrollPercent >= 75 && threshold == 75)` it would only fire/set that value once.

Reply:
Good point, I think that makes sense. Will update, thanks Mike.